python-sdk/nuscenes/tests/test_predict_helper.py

```diff
@@ -0,0 +1,503 @@
+# nuScenes dev-kit.
```

Comment:
I would recommend a local test folder inside the current dir (`python-sdk/nuscenes/predict`).

Reply:
Done!